website/docs/cloud-docs/workspaces/creating.mdx

```diff
@@ -56,11 +56,11 @@ For CLI and API-driven workspaces, the workspace overview page appears. For vers
 
 ### Configure Terraform Variables (Version Control Only)
 
-After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) and displays any that do not have a default value and are not defined in an existing global [variable set](/cloud-docs/api-docs/variable-sets). Terraform cannot perform successful runs in the workspace until you set values for these variables. 
+After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) and displays any that do not have a default value and are not defined in an existing global [variable set](/cloud-docs/api-docs/variable-sets). Terraform cannot perform successful runs in the workspace until you set values for these variables.
```

Comment:
yeah this is a tough one - I think this change might add more words that make the sentence more difficult to read. I might push back here and keep the way it was because I think it's easier to read even though it's passive.